dulwich/tests/test_client.py

```diff
@@ -23,6 +23,7 @@
 import sys
 import shutil
 import tempfile
+import mock
```

Comment:
Please just monkey patch, so we can avoid a dependency on mock just for these two tests.